src/ir_Mitsubishi.cpp

```diff
@@ -452,8 +454,104 @@ void IRMitsubishiAC::setMode(const uint8_t mode) {
       return;
   }
   _.Mode = mode;
+  // iSave10C can only be on in Heat mode.
+  if (mode != kMitsubishiAcHeat) {
+      setISave10C(false);
+  }
+}
+
+/// Set the iSave10C (i-SAVE) mode of the A/C.
+/// @param[in] state true, the setting is on. false, the setting is off.
+/// @note Normal minimum temp is 16C, and i-SAVE mode works as a gate to enable AC
```

Comment:
Linter issue:
`src/ir_Mitsubishi.cpp:465:  Lines should be <= 80 characters long  [whitespace/line_length] [2]`